main.go

```diff
@@ -1,9 +1,513 @@
-// Copyright 2017 The hubble-fit Authors.  All rights reserved.
+// Copyright 2017 The hubble-fit Authors. All rights reserved.
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
-
 // hubble-fit is a program to fit the Hubble diagram with a Cosmological model.
+
```

Comment:
no, actually, it's the line above that needed to be kept and this one to be dropped.
ie:

```go
// Copyright 2017 The hubble-fit Authors.  All rights reserved.
// Use of this source code is governed by a BSD-style
// license that can be found in the LICENSE file.

// hubble-fit is a program to fit the Hubble diagram with a Cosmological model.
package main

```